src/components/BurgerMenu/BurgerMenu.tsx

```diff
@@ -100,7 +104,14 @@ export const BurgerMenu: React.FC<Props> = ({
             to="favourites"
             className={handleBurgerButtonIsActive}
           >
-            <img src="./logos/favourites.svg" alt="logoFavourite" />
+            { currentURL !== 'favourites' ? <img
+              src="./logos/favourites.svg"
+              alt="logoFavourite"
+            />
+              : <img
+                src="./logos/favourites-selected.svg"
+                alt="logoFavouriteSelected"
+              />}
```

Comment:
make it spans, add classes(check in buttons.scss) and change it when it's active